src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -819,6 +847,17 @@ internal async Task<Collection<FunctionDescriptor>> GetFunctionDescriptorsAsync(
             return functionDescriptors;
         }
 
+        private static void UpdateFunctionMetadataLanguageForDotnetAssembly(IEnumerable<FunctionMetadata> functions, string workerRuntime)
+        {
+            foreach (var function in functions)
+            {
+                if (function.Language == DotNetScriptTypes.DotNetAssembly)
+                {
+                    function.Language = workerRuntime;
```

Comment:
Changing the function metadata language to match is enough to have customer apps running? In this misconfigured scenario does the app end up running like an inproc or isolated payload?

Reply:
Yes. The reason it works in 4.34 is we update the "Language" property of function metadata to "dotnet-isolated" (the language present in the workerconfig instance). Code [here](https://github.com/Azure/azure-functions-host/blob/ff0dea77c3de615a9a19f000595dc56262f47ad4/src/WebJobs.Script/Host/HostFunctionMetadataProvider.cs#L166) and [here](https://github.com/Azure/azure-functions-host/blob/ff0dea77c3de615a9a19f000595dc56262f47ad4/src/WebJobs.Script/Host/HostFunctionMetadataProvider.cs#L200-L204)). Having this change (Language in function metadata entries matching with FUNCTIONS_WORKER_RUNTIME
) and having at least one workerconfig instance to match the worker runtime value (dotnet-isolated) make things works.